testing/e2e/workload_test.go

```diff
@@ -56,6 +56,68 @@ var (
 
 func TestCreateFromGitWithAnnotations(t *testing.T) {
 	testSuite := it.CommandLineIntegrationTestSuite{
+		{
+			Name:         "Create workload and show emojis",
+			Skip:         runtime.GOOS == "windows",
+			WorkloadName: "test-create-workload-show-emojis",
+			Command: func() it.CommandLine {
+				c := *it.NewTanzuAppsCommandLine(
+					"workload", "create", "test-create-workload-show-emojis",
+					"--annotation=min-instances=2", "--annotation=max-instances=4",
+					"--app=test-create-workload-show-emojis",
+					"--git-tag=tap-1.2",
+					"--git-repo=https://github.com/sample-accelerators/spring-petclinic",
+					namespaceFlag,
+					"--type=web",
+				)
+				c.SurveyAnswer("y")
+				return c
+			}(),
+			Prepare: func(ctx context.Context, t *testing.T) (context.Context, error) {
+				var err error
+				if ctx, err = createPtyTerminal(ctx); err != nil {
+					t.Fatalf("error while opening pty %v", err)
+				}
+
+				return ctx, nil
+			},
+			ExpectedObject: &cartov1alpha1.Workload{
+				TypeMeta: workloadTypeMeta,
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-create-workload-show-emojis",
+					Namespace: it.TestingNamespace,
+					Labels: map[string]string{
+						"app.kubernetes.io/part-of":           "test-create-workload-show-emojis",
+						"apps.tanzu.vmware.com/workload-type": "web",
+					},
+				},
+				Spec: cartov1alpha1.WorkloadSpec{
+					Params: []cartov1alpha1.Param{{
+						Name: "annotations",
+						Value: v1.JSON{
+							Raw: []byte(`{"max-instances":"4","min-instances":"2"}`),
+						},
+					}},
+					Source: &cartov1alpha1.Source{
+						Git: &cartov1alpha1.GitSource{
+							URL: "https://github.com/sample-accelerators/spring-petclinic",
+							Ref: cartov1alpha1.GitRef{
+								Tag: "tap-1.2",
+							},
+						},
+					},
+				},
+			},
+			Verify: func(ctx context.Context, t *testing.T, output string, err error) {
+				if !emojisExistInOutput(output) {
```

Comment:
check is pretty generic. so it can have any emoji and it will pass right? can you make this more precise?